sql/wsrep_mysqld.cc

```diff
@@ -1044,7 +1044,8 @@ int wsrep_init() {
     }
 
     char ssl_opts[4096];
-    SslAcceptorContext::populate_wsrep_ssl_options(ssl_opts, sizeof(ssl_opts));
+    // TODO
```

Comment:
Can this be removed? or is there something left to do?

Reply:
Yes, removed.